### ISSUE TITLE
Logic about new reports showing?

### ISSUE THREAD

**Issue by yuliu** [mybb/mybb#3792]
Low priority, affecting both v1.8 & v1.9.

Sorry I haven't dig that much in previous code, but currently:

In AdminCP > Settings > General Configuration > Reported Content Notification Method (`reportmethod`), all its options are saying the reports will be "shown in the Mod CP". That's not very clear, since we have roughly three places to show: 

1. ModCP, of course.
1. AdminCP > home.
1. Notice bar site-wide in the front-end.

The most apparent issue is if we set this setting to `email` or `pms`, no **new reports** are shown in AdminCP's dashboard and front-end forum notice bar. That's should not be the origin of this setting.

I think it might not be well-designed or was not fully reviewd from previous upgrades with adding `email` and `pms` as options.
**To fix it, simple remove any `if($mybb->settings['reportmethod'] == "db")` in current code base, email or PM notifications will still work.**

As a refactoring to this setting in future versions, probably working with MyAlert or other alert system, it's better to remove this "db" option and make other two as two checkboxes attached in the alert system:
```
[   ] Email Mods on new report.
[ x ] PM Mods on new report.
```
Or a cascading settings:
```
[ X ] Notify/Alert Mods on new report. (since PM/Alert should be very basic features in a forum)
   -- [ X ] Email Mods as well.
```

**Comment by Eldenroot** [comment]
Good idea!

**Comment by Eldenroot** [comment]
I agree with the simple removing if($mybb->settings['reportmethod'] == "db") as a workaround. For 1.9/1.10 we should redesign the setings.

**Comment by Ben-MyBB** [comment]
https://github.com/mybb/mybb/issues/2537 should be taken in to account if we're looking at the report system overall.

**Comment by rajat315315** [comment]
Is anyone available for review? I think it is fixed.

**Comment by euantorano** [comment]
Hi,

We're currently in a PR freeze as we prepare to release 1.8.23. No PRs will be merged until that release is complete unless they are to fix critical bugs introduced in 1.8.23. Once the release cycle is finished, outstanding PRs will be reviewed and merged.
